unconstrained_test.go

```diff
@@ -1503,16 +623,5 @@ func testMinimize(t *testing.T, tests []UnconstrainedTest, method Method) {
 				}
 			}
 		}
-
```

Comment:
It is not deleted completely, almost every test is included twice, once from the standard location and once from a slightly perturbed minimum. Moreover, this test turns out to be the most difficult to pass, so just blindly run optimization from locations returned from minimumer.Minima() is not a kind idea, imo.